package.json

```diff
@@ -23,7 +23,6 @@
     "mqlight": "^1.0.0"
   },
   "devDependencies": {
-    "jscs": "^2.8.0",
     "eslint": "^1.10.3",
     "eslint-config-strongloop": "^1.0.2",
```

Comment:
It would be nice to eventually upgrade to `eslint-config-loopback: "^4.0.0"` (or whatever is the latest version).